src/qt/recentrequeststablemodel.cpp

```diff
@@ -234,7 +234,7 @@ bool RecentRequestEntryLessThan::operator()(const RecentRequestEntry& left, cons
     switch(column)
     {
     case RecentRequestsTableModel::Date:
-        return pLeft->date.toTime_t() < pRight->date.toTime_t();
+        return pLeft->date.toSecsSinceEpoch() < pRight->date.toSecsSinceEpoch();
```

Comment:
Out of scope, but I wonder why `bool QDateTime::operator<` is not used directly, without `toTimeT()/toSecsSinceEpoch()` ?